keyboards/sandortho/keymaps/default/keymap.c

```diff
@@ -0,0 +1,174 @@
+    ),
+    [_LOWER] = LAYOUT(
+   	KC_NO,     EXCL,     AT,      POUND,   DOLLAR,     PERCENT,   CARAT,        AND,        STAR,     LPAR,      RPAR,        KC_DELETE,               
+   	KC_NO,     KC_F1,    KC_F2,   KC_F3,   KC_F4,      KC_F5,     KC_F6,        KC_F7,      KC_F8,    KC_BSLASH, KC_LBRACKET, KC_RBRACKET,  
+   	KC_LSHIFT, KC_F9,    KC_F10,  KC_F11,  KC_F12,     KC_NO,     KC_NO,        KC_NO,      KC_NO,    KC_NO,     KC_NO,       KC_NO,  
+   	KC_NO,     KC_LCTRL, KC_LALT, KC_LGUI, KC_NO,      KC_NO,     KC_SPACE,     KC_NO,      KC_HOME,  KC_PGDOWN, KC_PGUP,     KC_END
+    )/*,
+    [_EMPTY] = LAYOUT(
+   	KC_NO,     KC_NO,    KC_NO,   KC_NO,   KC_NO,      KC_NO,     KC_NO,        KC_NO,      KC_NO,    KC_NO,     KC_NO,       KC_NO,  
+	KC_NO,     KC_NO,    KC_NO,   KC_NO,   KC_NO,      KC_NO,     KC_NO,        KC_NO,      KC_NO,    KC_NO,     KC_NO,       KC_NO,  
+	KC_NO,     KC_NO,    KC_NO,   KC_NO,   KC_NO,      KC_NO,     KC_NO,        KC_NO,      KC_NO,    KC_NO,     KC_NO,       KC_NO,  
+	KC_NO,     KC_NO,    KC_NO,   KC_NO,   KC_NO,      KC_NO,     KC_NO,        KC_NO,      KC_NO,    KC_NO,     KC_NO,       KC_NO,  
+    )*/
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case QMKBEST:
+            if (record->event.pressed) {
+                // when keycode QMKBEST is pressed
+                SEND_STRING("QMK is the best thing ever!");
+            } else {
+                // when keycode QMKBEST is released
+            }
+            break;
+        case QMKURL:
+            if (record->event.pressed) {
+                // when keycode QMKURL is pressed
+                SEND_STRING("https://qmk.fm/\n");
+            } else {
+                // when keycode QMKURL is released
+            }
+            break;
+	case EXCL:
+	    if (record->event.pressed) {
+	    	SEND_STRING("!");
+	    }else{
+	
+	    }
+	    break;
+	case AT:
+	    if (record->event.pressed) {
+	    	SEND_STRING("@");
+	    }else{
+	
+	    }
+	    break;
+	case POUND:
+	    if (record->event.pressed) {
+	    	SEND_STRING("#");
+	    }else{
+	
+	    }
+	    break;
+	case DOLLAR:
+	    if (record->event.pressed) {
+	    	SEND_STRING("$");
+	    }else{
+	
+	    }
+	    break;
+	case PERCENT:
+	    if (record->event.pressed) {
+	    	SEND_STRING("%");
+	    }else{
+	
+	    }
+	    break;
+	case CARAT:
+	    if (record->event.pressed) {
+	    	SEND_STRING("^");
+	    }else{
+	
+	    }
+	    break;
+	case AND:
+	    if (record->event.pressed) {
+	    	SEND_STRING("&");
+	    }else{
+	
+	    }
+	    break;
+	case STAR:
+	    if (record->event.pressed) {
+	    	SEND_STRING("*");
+	    }else{
+	
+	    }
+	    break;
+	case LPAR:
+	    if (record->event.pressed) {
+	    	SEND_STRING("(");
+	    }else{
+	
+	    }
+	    break;
+	case RPAR:
+	    if (record->event.pressed) {
+	    	SEND_STRING(")");
+	    }else{
+	
+	    }
+	    break;
```

Comment:
Just a heads up, most of these actually have dedicated keycodes. 

https://docs.qmk.fm/#/keycodes?id=us-ansi-shifted-symbols

And for the most part, they're just `LSFT(kc)`, actually.

```suggestion
```